lab_3_ann_retriever/start.py

```diff
@@ -30,7 +32,21 @@ def main() -> None:
     """
     with open("assets/secrets/secret_1.txt", "r", encoding="utf-8") as text_file:
         text = text_file.read()
-    result = None
+
+    documents = open_files()[0]
```

Comment:
do not call this function twice; return it from the one call

Reply:
done